zone/spells.cpp

```diff
@@ -6357,6 +6357,7 @@ void Client::SendItemRecastTimer(int32 recast_type, uint32 recast_delay)
 		ItemRecastDelay_Struct *ird = (ItemRecastDelay_Struct *)outapp->pBuffer;
 		ird->recast_delay = recast_delay;
 		ird->recast_type = static_cast<uint32>(recast_type);
+		ird->ignore_casting_requirement = ignore_casting_requirement; //True allows reset of item cast timers
```

Comment:
We need to rename one of these since they are the same, I suggest renaming the function argument. 

When we've named an input argument the same name as something else (local scoped or otherwise) it has caused unintended bugs

Reply:
Fixed and renamed to avoid this